bionic/code_hasher.py

```diff
@@ -62,28 +61,18 @@ class CodeHasher:
     """
 
     def __init__(self, suppress_warnings):
-        self._hash = hashlib.new("md5")
         self._suppress_warnings = suppress_warnings
         # This is used to detect circular references.
         self._object_depths_by_id = {}
+        self._hashes_by_id = {}
```

Comment:
It makes me a little nervous to be storing mutable hash objects in this cache. Would it make sense to store the final hash bytes instead?

Reply:
I did that at first, but then I realized I was using `hash_obj.digest()` to add to other hash objects, and `hash_obj.hexdigest()` to return the final result. But I think I can just use `str(hash_obj.digest())` instead of `hexdigest` to return the final result instead.